source/Halibut.Tests/BackwardsCompatibility/InvocationExceptionsAreMappedCorrectlyFixture.cs

```diff
@@ -20,11 +20,7 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                        .WithStandardServices()
                        .Build(CancellationToken))
             {
-                var echo = clientAndService.CreateAsyncClient<IEchoService, IAsyncClientEchoService>(se =>
```

Comment:
Non need to lower these numbers here, it was being flaky locally with a high level of parallel tests.